powershell-adapter/psDscAdapter/psDscAdapter.psm1

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+$script:CurrentCacheSchemaVersion = 1
```

Comment:
did you consider using `[version]` type?

Reply:
Seems like an overkill for this simple case here;
Besides, the cache does not have major/minor/build concepts.